Marlin/ultralcd.cpp

```diff
@@ -2799,16 +2804,17 @@ void lcd_update() {
 
         #if ENABLED(DOGLCD)  // Changes due to different driver architecture of the DOGM display
           static int8_t dot_color = 0;
-          dot_color = 1 - dot_color;
-          u8g.firstPage();
-          do {
+          if (!drawing_screen) {
+            u8g.firstPage();
+            dot_color = 1 - dot_color;
+          }
             lcd_setFont(FONT_MENU);
             u8g.setPrintPos(125, 0);
             u8g.setColorIndex(dot_color); // Set color for the alive dot
             u8g.drawPixel(127, 63); // draw alive dot
             u8g.setColorIndex(1); // black on white
```

Comment:
The 'Alive Dot' is cool...   But I'm not sure it really serves a purpose.   If the printer is printing...  The numbers on the screen are changing and that little dot blinking just wastes CPU cycles.   If the printer is idle, I can certainly click the encoder wheel to see if the LCD Panel is frozen.

My vote would be to get rid of the blinking dot.  But if some people insist it be there, give people that don't want it the option of turning it off and saving CPU cycles.